gexec/build_test.go

```diff
@@ -60,28 +63,42 @@ var _ = Describe(".BuildWithEnvironment", func() {
 
 var _ = Describe(".BuildIn", func() {
 	var (
-		gopath string
+		original string
+		gopath   string
+		target   string
 	)
 
 	BeforeEach(func() {
-		gopath = os.Getenv("GOPATH")
-		Expect(gopath).NotTo(BeEmpty())
-		Expect(os.Setenv("GOPATH", "/tmp")).To(Succeed())
-		Expect(os.Environ()).To(ContainElement("GOPATH=/tmp"))
+		original = os.Getenv("GOPATH")
+		gopath = filepath.Join(os.TempDir(), "gopath1")
+		target = "github.com/onsi/gomega/gexec/_fixture/firefly/"
+		Expect(os.MkdirAll(filepath.Join(gopath, "src", target), 0755)).To(Succeed())
```

Comment:
Would it be possible to use temporary directories rather than hard-coded `gopath1`?  The reason is that there would be interference if two people on the same server were to run these tests at the same time.  I know it's unlikely, but it would be really hard to debug if it happened...